src/App.test.js

```diff
@@ -715,4 +716,50 @@ describe('App component', () => {
       expect(localStorageItems['dopplerFirstOrigin.value']).toBeUndefined();
     });
   });
+
+  describe('activation in progress parameter', () => {
+    it('should call google adwords when query string contains `activationInProgress%20=%20true`', () => {
+      // Arrange
+      const dependencies = {
+        window: {
+          gtag: jest.fn(),
+          Menubutton: MenubuttonFake,
+        },
+      };
+
+      // Act
+      render(
+        <AppServicesProvider forcedServices={dependencies}>
+          <Router initialEntries={['/login?activationInProgress%20=%20true']}>
+            <App locale="en" />
+          </Router>
+        </AppServicesProvider>,
+      );
+
+      // Assert
+      expect(dependencies.window.gtag).toBeCalledTimes(1);
+    });
+
+    it('should not to be call google adwords', () => {
+      // Arrange
+      const dependencies = {
+        window: {
+          gtag: jest.fn(),
+          Menubutton: MenubuttonFake,
+        },
+      };
+
+      // Act
+      render(
+        <AppServicesProvider forcedServices={dependencies}>
+          <Router initialEntries={['/login']}>
+            <App locale="en" />
+          </Router>
+        </AppServicesProvider>,
+      );
+
+      // Assert
+      expect(dependencies.window.gtag).not.toBeCalled();
+    });
+  });
```

Comment:
Could you please add some more scenarios:

* [x] google adwords should be called when query string contains `activationInProgress%20=%20true`.

    Done.

* [x] google adwords should not be called when query string does not contain `activationInProgress` parameter.

    Done.

* [ ] google adwords should be called when query string contains `activationInProgress=true`.

    `/login?activationInProgress=true`

* [ ] google adwords should not be called when query string `activationInProgress` parameter is not `true`.

    `/login?activationInProgress=another`

* [ ] google adwords should not be called when query string `activationInProgress` has no value.

    `/login?activationInProgress`

Reply:
ready